packages/ketcher-core/src/application/editor/data/monomers.ket

```diff
@@ -86965,6 +86974,706 @@
       }
     ],
     "naturalAnalogShort": "F"
+  },
+   "monomerTemplate-NMethylProline___NMethylProline": {
+    "type": "monomerTemplate",
+    "atoms": [
+            {
+                "label": "N",
+                "location": [
+                    5.7425081950194015,
+                    -7.7752604484558105,
+                    0
+                ]
+            },
+            {
+                "label": "C",
+                "location": [
+                    6.608533598803841,
+                    -7.2752604484558105,
+                    0
+                ],
+                "stereoLabel": "abs"
```

Comment:
The structure here seems to be different comparing to what's attached in the requirements
<img width="758" alt="image" src="https://github.com/user-attachments/assets/41e754c1-12d2-466f-adae-58fca87e856a">
This will lead to incorrect leaving groups definition if you will use what's described in ticket

Could you please double-check that structure is correct for this amino-acid as well as for the other two